tests/test_csv_reader.py

```diff
@@ -29,6 +29,25 @@ def csv(tmp_path):
     return p
 
 
+@pytest.fixture
+def csv_bad(tmp_path):
+    content = [
+        "# Date (yyyy-mm-ddTHH:MM),2023-03-06T11:00,,,",
+        "#Name field campaign,SnowEx 2023,,,",
+        "#UTM_Zone,6W,,,",
+        "#Easting,466153,,,",
+        "#Northing,7193263,,,",
+        "#Timing,25 min,,,",
+    ]
+
+    d = tmp_path / __name__
+    d.mkdir()
+    p = d / "test.csv"
```

Comment:
@eigenbeam does it matter that both the good and bad fixtures write to the same file?